### ISSUE TITLE
Adaptive dropout rate by custom callback

### ISSUE THREAD

**Issue by dervishson** [dervishson/Deep-RL#1]
Hi all,
I would like to create my own custom callback for dropout rate. And my code is seen below:

_from keras import backend
from keras.callbacks import Callback_

class MyDropCallback(Callback):
    def __init__(self,dropping_rate):
        self.dropping_rate=dropping_rate
    
    def updating_mydroprate(self,epoch,dropping_rate):
        if epoch==35:
            dropping_rate=0.65
        elif epoch==60:
            dropping_rate=0.75
        elif epoch==80:
            dropping_rate=0.8
    
    def on_epoch_begin(self,epoch,logs=None):
        dropping_rate=self.updating_mydroprate(epoch,self.dropping_rate)
        **backend.set_value(self.model.?????,,dropping_rate)**_

At the end line of the code above, i am not sure what to write at the place ????. For example if there is a learning rate schedule of a custom callback, in the last line i would write (self.model.optimizer.lr,lr). However in my dropout code, i am not sure what to write?
Thanks for helps.
